pages/privacy-policy.tsx

```diff
@@ -0,0 +1,399 @@
+            </li>
+            <li>
+              <p className="flex items-center gap-1 py-4 text-sm font-medium text-stone-800">
+                Last updated{" "}
+                <span className="inline-flex">
+                  <Pill>August 1, 2023</Pill>
+                </span>
+              </p>
+            </li>
+          </ul>
+        </aside>
+        <main
+          className={`
+          space-y-8
+        `}
+        >
+          <article
+            className={`
+              space-y-4
+            `}
+          >
+            <HashAnchorTarget
+              id="information-we-collect"
+              classNames="space-y-2"
+            >
+              <h2
+                className={`
+                text-5xl
+                font-semibold
+              `}
+              >
+                Information we collect
+              </h2>
+            </HashAnchorTarget>
+            <p>
+              Our collection and use of personal data varies depending on the
+              context of your interactions with us. We collect information that
+              you provide to us directly, such as when you join the directory or
+              communicate with us.
+            </p>
+            <p>
+              While public information will be generally available on the
+              website,{" "}
+              <strong>we won’t mass distribute or sell this information</strong>{" "}
+              without your consent.
+            </p>
+            <HashAnchorTarget id="joining-the-directory" classNames="space-y-2">
+              <h3 className="text-3xl font-semibold">Joining the Directory</h3>
+              <p>
+                When you join the directory, we collect{" "}
+                <strong>
+                  your name, location, aspects of your professional career, and
+                  other public information
+                </strong>{" "}
+                you choose to provide. This information may be displayed and be
+                used to filter the directory.
+              </p>
+              <p>
+                We take the stance that{" "}
+                <strong>your email address is private information</strong>. We
+                will not share (or sell) it without your explicit consent. Only
+                trusted members of our administrative hui will have access to
+                this contact information.
+              </p>
+              <p>
+                On joining the directory, we assign you a{" "}
+                <strong>unique ID</strong>. This ID helps us recognize you when
+                you visit the site later.
+              </p>
+              <p>
+                We also <strong>track metrics of visitor events</strong> related
+                to the directory, like click counts of the link you provided us.
+                Please{" "}
+                <Link
+                  href="mailto:kekai@hawaiiansintech.org,kamakani@hawaiiansintech.org"
```

Comment:
Added a new `generateAdminMailToLink()` function that accepts optional subject and body parameters for future admin email links.

I also removed the Hackathonʻs contact section since itʻs no longer relevant. 👍